ubyssey/static/src/styles/modules/events/_calendar.scss

```diff
@@ -0,0 +1,178 @@
+$border                           : 1px dotted $color-border-gray-dark;
+$calendar-event-time-label-color  : #006fcc;
+
+$tiles-margin                     : 35px;
+
+#calendar {
```

Comment:
We try to avoid using IDs for css targeting. Can you do `.c-event-calendar` instead?